library/network_connections.py

```diff
@@ -1004,6 +1004,11 @@ def connection_create(self, connections, idx, connection_current=None):
                     Util.path_to_glib_bytes(connection["ieee802_1x"]["ca_cert"]),
                 )
 
+            if connection["ieee802_1x"]["ca_path"]:
+                s_8021x.set_property(
+                    NM.SETTING_802_1X_CA_PATH, connection["ieee802_1x"]["ca_path"],
```

Comment:
As mentioned here:  https://github.com/linux-system-roles/network/issues/229#issuecomment-629760274 NetworkManager doesn't actually set this property properly. Fairly sure this is a bug.

Reply:
NetworkManager got fixed upstream. I adjusted the module/tests to check of support and fail if it is requested but missing.